src/types/CoverageIntegration.ts

```diff
@@ -0,0 +1,25 @@
+import type { CoverageIntegrationRun } from './CoverageIntegrationRun';
+import type { NewsroomRef } from './Newsroom';
+
+type Iso8601DateTime = string;
+
+export enum PROVIDER {
```

Comment:
We already have Provider enum in CoverageEntry. It should probably be re-used (with Google Alerts added)

https://github.com/prezly/javascript-sdk/blob/master/src/types/CoverageEntry.ts#L66-L76

Reply:
Good remark 👍🏻  Done in https://github.com/prezly/javascript-sdk/pull/315/commits/397d0b4d86ad6078947ab7900e9becd24b5f366f